OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -57,12 +58,33 @@ public class CloakInfo : PausableConditionalTraitInfo
 		public readonly string Palette = "cloak";
 		public readonly bool IsPlayerPalette = false;
 
-		public readonly BitSet<CloakType> CloakTypes = new BitSet<CloakType>("Cloak");
+		public readonly BitSet<DetectionType> DetectionTypes = new BitSet<DetectionType>("Cloak");
 
 		[GrantedConditionReference]
 		[Desc("The condition to grant to self while cloaked.")]
 		public readonly string CloakedCondition = null;
 
+		[Desc("The type of cloak. Same type of cloaks won't tigger cloaking and uncloaking sound and effect.")]
```

Comment:
`tigger`

Reply:
How about the other usecase of stealth apc?

Reply:
Fixed and dropped